### PR TITLE
Use Project.toml and drop support for pre-1.0

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

**Comment by vchuravy** [comment]
replaces #41 and fixes #36

**Comment by vchuravy** [comment]
I also added registrator and tagbot

File: .travis.yml [hunk 1]
```diff
@@ -24,8 +24,8 @@ notifications:
 #  - julia -e 'Pkg.clone(pwd()); Pkg.build("Example"); Pkg.test("Example"; coverage=true)';
 
 after_success:
-  - julia -e 'if VERSION >= v"0.7.0-" using Pkg end; cd(Pkg.dir("Example")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
-#  - julia -e 'if VERSION >= v"0.7.0-" using Pkg end; cd(Pkg.dir("Example")); Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())';
+  - julia -e 'using Pkg; cd(Pkg.dir("Example")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
```
Comment (KristofferC) [inline .travis.yml]: Can remove `cd`

File: .travis.yml [hunk 2]
```diff
@@ -24,8 +24,8 @@ notifications:
 #  - julia -e 'Pkg.clone(pwd()); Pkg.build("Example"); Pkg.test("Example"; coverage=true)';
```
Comment (KristofferC) [inline .travis.yml]: This is not true anymore.

Comment (KristofferC) [reply .travis.yml]:
```jl
julia --project -e 'import Pkg; Pkg.build(); Pkg.test(; coverage=true)
```

File: test/runtests.jl [hunk 3]
```diff
@@ -1,9 +1,4 @@
-using Example
-if VERSION < v"0.7.0-DEV.2005"
-    using Base.Test
-else
-    using Test
-end
+using Test
```
Comment (fredrikekre) [inline test/runtests.jl]:
```suggestion
using Test, Example
```

File: .travis.yml (hunk 2 above)
Comment (vchuravy) [reply .travis.yml]: Thanks!

**Pull request merged**